src/pages/_layout-dev/style.module.css

```diff
@@ -0,0 +1,48 @@
+/* DEMO CONTENT */
```

Comment:
As mentioned in accompanying `.tsx` file, intention is to delete this once we're done reviewing and have the layout work approved. Though also considering trying to move this into Swingset or something (though not exactly sure how I'll do that, yet).